paddle/fluid/framework/scope.cc

```diff
@@ -59,6 +59,8 @@ Scope& Scope::NewScope() const {
   return *child;
 }
 
+Scope* Scope::NewTmpScope() const { return new Scope(this); }
```

Comment:
`NewTmpScope ` should return a `unique_ptr` at least.

Reply:
Done